debian/snapd.postrm

```diff
@@ -33,17 +43,17 @@ if [ "$1" = "purge" ]; then
             rm -f "/snap/bin/$snap"
             rm -f "/snap/bin/$snap".*
             # snap mount dir
-            umount -l "/snap/$snap/$rev" 2> /dev/null || true
+            umount -d -l "/snap/$snap/$rev" 2> /dev/null || true
```

Comment:
Interesting, I  didn't know about `-d`. Is this something we should be doing ourselves elsewhere?